src/i18n/Base.js

```diff
@@ -444,8 +452,8 @@ export default {
     en: 'Default',
   },
   defaultPwdTip: {
-    zh: '登录成功！检测到您正在使用默认密码登录，请及时修改密码',
-    en: 'Login successful! Detected that you are logging in with the default password, please change your password in time.',
+    zh: '检测到您使用默认密码登陆, 为了您的安全请修改密码：',
+    en: 'Detected that you are logging in with the default password. For your safety, please change your password:',
```

Comment:
```suggestion
    en: 'Logging in with the default password has been detected. For account security, a password change is strongly recommended:',
```

Reply:
maybe "To ensure your of account security, we kindly request you to change your default password."